kubernetes-client/src/main/java/io/fabric8/kubernetes/client/dsl/base/BaseOperation.java

```diff
@@ -124,13 +124,27 @@ protected BaseOperation(OkHttpClient client, Config config, String apiGroup, Str
    */
   private static <T> String apiVersion(T item, String apiVersion) {
     if (apiVersion != null && !apiVersion.isEmpty()) {
-      return apiVersion;
+      return trimVersion(apiVersion);
     } else if (item instanceof HasMetadata) {
-      return ((HasMetadata)item).getApiVersion();
+      return trimVersion(((HasMetadata)item).getApiVersion());
     }
     return null;
   }
 
+  /**
+   * Separates apiVersion for apiGroup/apiVersion combination.
+   * @param apiVersion  The apiVersion or apiGroup/apiVersion combo.
+   * @return            Just the apiVersion part without the apiGroup.
+     */
+  private static String trimVersion(String apiVersion) {
+    if (apiVersion == null) {
+      return null;
+    } else {
+      String[] versionParts = apiVersion.split("/");
+      return versionParts.length > 1 ? versionParts[versionParts.length - 1] : apiVersion;
```

Comment:
You don't need to do that check, just always return `versionParts[versionParts.length - 1]` - that's what I was trying to say before! See https://repl.it/DfTr/0 for proof.